### PR TITLE
Create namespace with references for modules

### PR DESCRIPTION
**Pull request by zherczeg**

### CODE REVIEW

File: tests/jerry/es.next/module-circular-02.mjs [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as o from "module-circular-01.mjs"
+
+if (o.early != "Loaded")
```
Comment (zherczeg) [inline tests/jerry/es.next/module-circular-02.mjs:17]: Is it possible to not run a test in this directory? Not really a problem if not, just asking. Perhaps adding a bad extension, e.g.  `._mjs`

**Comment by zherczeg** [comment]
This patch is ready, please check.

File: jerry-core/ecma/base/ecma-helpers.c [hunk 2]
```diff
@@ -1008,6 +1066,31 @@ ecma_set_named_accessor_property_setter (ecma_object_t *object_p, /**< the prope
 #endif /* JERRY_CPOINTER_32_BIT */
 } /* ecma_set_named_accessor_property_setter */
 
+#if JERRY_MODULE_SYSTEM
+
+extern inline ecma_property_value_t * JERRY_ATTR_ALWAYS_INLINE
+ecma_get_property_value_from_named_reference (ecma_property_value_t *reference_p) /**< data property reference */
```
Comment (rerobika) [inline jerry-core/ecma/base/ecma-helpers.c:1072]: Missing doc.

**Review by rerobika — APPROVED** [review]
LGTM after this minor issue.

**Review by dbatyai — APPROVED** [review]
LGTM

**Pull request merged**